R/upgrade.R

```diff
@@ -54,14 +54,15 @@ renv_upgrade_impl <- function(project, version, reload, prompt) {
 
   reload <- reload %||% renv_project_loaded(project)
 
-  old <- renv_snapshot_description(package = "renv")
+  lockfile <- renv_lockfile_load(project)
```

Comment:
I don't know if this is correct or not, but `renv_snapshot_description()` returns a rather different data structure so the comparison below was always `FALSE`.